lib/browser/api/web-contents.js

```diff
@@ -145,12 +145,20 @@ const webFrameMethods = [
   'insertCSS',
   'insertText',
   'setLayoutZoomLevelLimits',
-  'setVisualZoomLevelLimits'
+  'setVisualZoomLevelLimits',
 ]
 
-const asyncWebFrameMethods = function (requestId, method, callback, ...args) {
+/**
+ * @this WebContents
+ * @param {number} requestId
+ * @param {string} method
+ * @param {Function} callback
+ * @param  {...any} args
+ * @return {Promise<any>}
```

Comment:
i'll stop putting comments on these odd doc-comments now

Reply:
This function requires the `@this` JS Doc comment, and there is another rule requiring all JS Doc comments to be valid and complete.  So if I add `@this` I also needed to add all the other `@blah` things